docs/route/meta.md

```diff
@@ -148,13 +148,13 @@ export const meta: MetaFunction<
 
 ## Gotchas with `meta` and Nested Routes
 
-Because multiple nested routes render at the same time, there is some merging that needs to happen to determine the meta tags that ultimately render. Remix gives you complete control over this merge because there is no obvious default.
+Because multiple nested routes render at the same time, there is some merging that needs to happen to determine the meta-tags that ultimately render. Remix gives you complete control over this merge because there is no obvious default.
```

Comment:
```suggestion
Because multiple nested routes render at the same time, there is some merging that needs to happen to determine the meta tags that ultimately render. Remix gives you complete control over this merge because there is no obvious default.
```